api4/team_test.go

```diff
@@ -464,6 +464,30 @@ func TestUpdateTeamMemberRoles(t *testing.T) {
 	CheckNoError(t, resp)
 }
 
+func TestUpdateTeamDescription(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+
+	team := &model.Team{DisplayName: "Name", Description: "Some description", Name: "z-z-" + model.NewId() + "a", Email: "success+" + model.NewId() + "@simulator.amazonses.com", Type: model.TEAM_OPEN}
+	team, _ = Client.CreateTeam(team)
+
+	team.Description = "updated description"
+	uteam, resp := Client.UpdateTeam(team.Id, team);
+	CheckNoError(t, resp)
+
+	if uteam.Description != "updated description" {
+		t.Fatal("Update failed")
+	}
```

Comment:
Can you add more test cases checking to make sure that we can only update the allowed fields?